packages/stellar-axelar-std/src/string.rs

```diff
@@ -0,0 +1,25 @@
+#[cfg(feature = "alloc")]
+use alloc::vec;
+#[cfg(feature = "alloc")]
+use alloc::vec::Vec;
```

Comment:
```suggestion
#![cfg(any(test, feature = "alloc"))]

extern crate alloc;

use alloc::vec::{self, Vec};
```
apply the cfg for the whole file instead?